File: c3p0demo/src/main/scala/poso/Book.scala
package poso


import reflect.BeanProperty

/**
 * Created by IntelliJ IDEA.
 * User: Administrator
 * Date: 2009-12-29
 * Time: 8:10:07
 * To change this template use File | Settings | File Templates.
 */

class Book extends AbstractProduct {
  @BeanProperty
  var author: String = _
  @BeanProperty
  var description: String = _
}

File: jsonrpcdemo/src/main/java/com/ggd543/service/Book.java
package com.ggd543.service;

/**
 * Created with IntelliJ IDEA.
 * User: Administrator
 * Date: 13-10-2
 * Time: 下午5:58
 * To change this template use File | Settings | File Templates.
 */
public class Book {
    public int cn;
    public String name;
    public User author;

    public Book() {
    }

    public Book(int cn, String name, User author) {
        this.cn = cn;
        this.name = name;
        this.author = author;
    }
}
